bumble/profiles/asha_service.py

```diff
@@ -101,49 +104,59 @@ def on_audio_control_point_write(connection: Connection, value):
                     )
                 )
 
+        def on_read_only_properties_read(connection: Connection) -> bytes:
+            value = (
```

Comment:
is this value expected to change during the lifetime of the characteristic? If not, why not construct it just once at init time?